Metalama.Framework.Engine/Pipeline/AspectPipeline.cs

```diff
@@ -456,6 +456,7 @@ private static ImmutableArray<AdditionalCompilationOutputFile> GetAdditionalComp
                 additionalCompilationOutputFiles: additionalCompilationOutputFiles );
 
             var allAspects = Enumerable.Empty<AspectInstanceResult>();
+            var allValidators = Enumerable.Empty<ValidatorInstance>();
```

Comment:
Collecting all declaration validators is useless. Propagating a flag `hasDeclarationValidator` is enough.